shopify-eventbridge-lambda/shopify-eventbridge-lambda/README.md

```diff
@@ -0,0 +1,114 @@
+    ```
+    cdk deploy --parameters shopifyEventBusName=SAAS_EVENT_BUS_NAME_HERE
+    ```
+
+4. Note the outputs from the CDK deployment process. These contain the resource names and/or ARNs which are used for testing. This stack will output the name of the Lambda function deployed for testing to the CLI. See the example below. 
+
+```
+Outputs:
+ShopifyIntegrationStack.ShopifyProcessFailedLoginLambdaOutput = ShopifyIntegrationStack-ShopifyProcessFailedLoginLambd....
+```
+
+## How it works
+
+This service interaction uses an existing Shopify SaaS integration in the customer's AWS account. If you do not have the Shopify SaaS integration set up in your AWS account, please set it up before deploying this pattern. View the [integration on Shopify's developer site](https://shopify.dev/docs/apps/build/webhooks/configuration/eventbridge#eventbridge-payload-structure).
+
+This pattern demonstrates how to:
+1. Write EventBridge rules that match Shopify's event pattern
+2. Send events from Shopify's EventBridge webhook integration to Amazon CloudWatch for logging and debugging
+3. Transform Shopify events using AWS Lambda and allows you to connect to other services going forward
+
+See the below architecture diagram from the data flow of this pattern. 
+
+![Architecture Diagram](./img/readme-arch-diagram.png)
+
+## Testing
+
+### Test the AWS Lambda Function
+
+The event.json file included in this pattern is a sample EventBridge event from Shopify. This event can be used to test the Lambda function and EventBridge rules deployed by this pattern.
+
+To test the Lambda function via the CLI, copy and paste the following command, replacing the variables in <> with your own values:
+```
+aws lambda invoke --function-name <ShopifyIntegrationStack-ShopifyProcessProductUpdatedLambd-....> --payload file://event.json --cli-binary-format raw-in-base64-out response.json
+```
+
+You should receive a response that looks like: 
+```
+{
+    "StatusCode": 200,
+    "ExecutedVersion": "$LATEST"
+}
+```
+
+The command should have create a response.json file in your directory. If you open this file, you see the output of the Lambda function.  
+
+### Test the EventBridge Rule
+
+You cannot put events on the Shopify event bus, only Shopify can publish events to this event bus. To test that the EventBridge rules deployed by this pattern were successfully deployed, follow these instructions: 
+
+1. Navigate to the Amazon EventBridge console. Select "Rules". 
+
+2. From the Event bus list, choose the SaaS event bus associated with your Shopify partner event source. 
+
+3. From the Rules list, select the "ShopifyIntegrationStack-ShopifySpecficEventsRule54D..." 
```

Comment:
Nit: Could you include more chars in the rule name or make it more clear to select this rule? As the rule name prefixes are same it could trick into going through another rule.